composer.json

```diff
@@ -9,10 +9,11 @@
         "symfony/polyfill-php72": "*",
         "symfony/polyfill-php73": "*",
         "symfony/polyfill-php74": "*",
-        "symfony/polyfill-php80": "*"
+        "symfony/polyfill-php80": "*",
```

Comment:
both can be removed as now the php version is forced via the require

Reply:
I'm not sure ... all these are in the `replace` section of `composer.json`. If I'm right, listing them here means that Composer won't download those packages even if some dependency requires them. So, the idea is that you add new polyfills when you upgrade your minimum required PHP version.

Reply:
ho right! I did not pay attention where the location were at first :s my bad